scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -171,32 +275,27 @@ library_ids <- c(
   "SCPCL000491"
 )
 
-# subset sce to only contain samples with library IDs of interest 
+# subset sce to only contain samples with IDs of interest 
 samples_to_keep <- integrated_sce$sample %in% library_ids
 rms_sce <- integrated_sce[,samples_to_keep]
 
-# remove original integrated object 
+# remove original integrated object to save some memory
 rm(integrated_sce)
-```
 
-```{r}
-# we are left with 3 from each subdiagnosis 
-sample_metadata %>%
-  dplyr::filter(library_id %in% library_ids) %>%
-  dplyr::count(subdiagnosis)
+# print out our new sce 
```

Comment:
I have been capitalizing SCE when not in a variable name or similar. I just noticed this one, but we probably want to be consistent.
```suggestion
# print out our new SCE 
```